plugins/jira-dashboard-backend/src/api.ts

```diff
@@ -58,7 +58,11 @@ export const getIssuesByFilter = async (
   const issues: Issue[] = [];
   for (const project of projects) {
     const { projectKey, instance } = project;
-    const jql = jqlQueryBuilder({ project: [projectKey], components, query });
+    const jql = jqlQueryBuilder({
+      project: ['"' + projectKey + '"'],
```

Comment:
Maybe a template string makes most sense here?

```js
      project: `"${projectKey}"`
```

Reply:
It may be better to do it inside the "jqlQueryBuilder" function? We already escape the "components" argument in the same way. There is other uses of the "jqlQueryBuilder" as  well.

Reply:
Ok! Modified that jql builder function to support it.

Reply:
We were using single qoutes for escaping whitespaces in the "component" query. Can't we use the same for projects?

Reply:
Ok - should updateIssuesByComponent also be updated to use the JQLBuilder as well?

https://github.com/AxisCommunications/backstage-plugins/blob/eb13a1d82d941a7ef4e8f9c6e9b515d3e0cbe0d9/plugins/jira-dashboard-backend/src/api.ts#L146